src/XeroPHP/Remote/Query.php

```diff
@@ -42,22 +60,63 @@ public function from($class) {
     }
 
     /**
+     * @param string $where
+     * @param mixed|null $value
      * @return $this
      */
-    public function where() {
-        $args = func_get_args();
-
-        if(func_num_args() === 2) {
-            $this->where[] = sprintf('%s=="%s"', $args[0], $args[1]);
+    public function where($where, $value = null) {
+        if ($value !== null) {
+            $parameterName = $this->addInternalParameter($value);
+            $this->where[] = sprintf('%s==%s', $where, ":$parameterName");
```

Comment:
@calcinai I was also not sure with the sha1 for parameter code. I used `internalParameters` instead. Anything added via where with value argument will be added to internal parameter list and then translated to final value.

We have to do this, because otherwise value that starts with colon could get translated to parameter value.